SQL/0000-00-00-schema.sql

```diff
@@ -968,7 +972,7 @@ CREATE TABLE `mri_violations_log` (
   `CandID` int(6) DEFAULT NULL,
   `Visit_label` varchar(255) DEFAULT NULL,
   `CheckID` int(11) DEFAULT NULL,
-  `Scan_type` int(11) unsigned DEFAULT NULL,
+  `MriScanTypeID` int(11) unsigned DEFAULT NULL,
```

Comment:
@nicolasbrossard @driusan I am curious what you think of the `DEFAULT NULL` for `MriScanTypeID` of table `mri_violations_log`. My understanding is that it can never be `NULL` based on the code since the violations log happen after scan identification. But before we remove the `DEFAULT NULL` from this table, I'd like your input. :)

Thank you!

Reply:
@cmadjar I agree with you: I don't think a ` DEFAULT NULL` makes sense here so I'd remove it.

Reply:
@cmadjar Do you want me to add a `NOT NULL` constraint then ? It could be in this (already quite large) PR or another later.

Reply:
That's fine, it can be in a different PR. @maximemulder Could you create an issue for it so we do not forget about it?